<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>526</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>526</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>992</width>
    <height>526</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCliente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opción</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cliente 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cliente 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cliente 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cliente 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cliente 4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnRegistrarVenta">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2ABF9E;
    color: white;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:pressed {
 background-color: #239D81;
}</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>971</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 55, 59);
border-radius: 6px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Hv BT</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(96, 206, 218);</string>
    </property>
    <property name="text">
     <string>Registro de Ventas - POS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/imagenes.qrc">:/xy/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Combustible: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Litros</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Historial de ventas</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateVenta">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::Section::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2025</year>
     <month>9</month>
     <day>9</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>Nota: usar punto como sepaador decimal (.)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnularVenta">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFba6a;   /* naranja normal */
    color: white;
    border-radius: 10px;
    padding: 10px;
}
QPushButton:pressed {
    background-color: #D4881C;   /* naranja más oscuro al presionar */
}
</string>
   </property>
   <property name="text">
    <string>Anular</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtCombustible">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>Valor automático</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método de pago</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMetodoPago">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opción</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Efectivo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yape</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Visa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otros</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbSurtidor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>--- Seleccione una opción</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanque 1 - Petroleo </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanque 2 - Petroleo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanque 3 - Diesel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanque 4 - Regular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanque 5 - Premiun</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Surtidor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLitros">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monto total (S/.)</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblVenta">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>590</width>
     <height>231</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>590</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget::item:selected {
    background-color: rgb(51, 55, 59);  /* el mismo turquesa que en el header */
    color: white;                         /* texto blanco para contraste */
}
QHeaderView::section {
    background-color: rgb(96, 206, 218); /* fondo turquesa */
                      /* texto oscuro para buen contraste */
    font-weight: bold;
    padding: 6px;
    border: 1px solid #CCCCCC;
}

QTableWidget {
    gridline-color: #444;
    background-color: #FFFFFF;  /* fondo de la tabla (oscuro) */
    color: #33373B;             /* texto claro */
	border-radius: 8px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
   </property>
   <property name="tabKeyNavigation">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::TextElideMode::ElideRight</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::PenStyle::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>72</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>91</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>id_venta</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cliente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Surtidor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pago</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total (S/.)</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>123</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>3456789</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>12/12/2024</string>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>Yape</string>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>24.5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hora:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeVenta">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImprimirBoleta">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/imagenes.qrc"/>
  <include location="../resources/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
